src/index.js

```diff
@@ -1 +1,28 @@
-console.log('minimalist');
\ No newline at end of file
+import './style.css';
+import { li } from '../modules/List.js';
+
+const TODOS = [
+  {
+    description: 'Washing the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
```

Comment:
- [ ] **Required:** You have done an amazing job so far. However, your list should be loaded in the order of index.

**Hint:** `Array.sort((a, b) => a.index - b.index);`